server/bin/pbench-server-set-result-state

```diff
@@ -5,14 +5,28 @@
 opts=$SHELLOPTS
 case $opts in
     *xtrace*)
-        dir=$(dirname $(which $0))
-        PROG=$(basename $(which $0))
+        dir=$(dirname $(command -v $0))
+        PROG=$(basename $(command -v $0))
         ;;
     *)
         dir=$(dirname $0)
         PROG=$(basename $0)
         ;;
 esac
+case $# in
+    2)
+        :
+        ;;
+    *)
+        echo "Usage: $PROG <satellite-config> <output-tar-ball>" >&2
+        exit 1
+        ;;
+esac
+
+if [[ -z "${dir}" || -z "${PROG}" ]]; then
```

Comment:
Shouldn't this be checked after line 15 above since `$PROG` is referenced in the above `case` construct?